### ISSUE TITLE
Credentials

### ISSUE THREAD

**Issue by patrickboulay** [EddyVerbruggen/cordova-plugin-googleplus#395]
Hi,

Im not sure how work the credentials but I tried to create one as a OAuth Client Id -> Web Application and is not working.

Also, I tried with my credential type Android but I cant create one for debug because I already have one for release and we can not have 2 credentials with the same package name.

For this plugin, what kind of credential I need to create and have information I need to provide them. Its a OAuth Client Id right? and wich type (Web Application, Android)

Also, if I understand well, the webClientId that I provide to window.plugins.googleplus.login is the Client Id?

Thanks and sorry to be stupid on this part :D

**Comment by amit777** [comment]
Were you able to fix your issue?  Any pointers?

**Comment by patrickboulay** [comment]
No, I switch to firebase authentification because I was not able to find a
solution... But I will prefer to use a native plugin because using the web
interface of firebase is not nice..

**Comment by patrickboulay** [comment]
its ok, you can close the issue.. now I have an error 10 so I think im not
too far :)
